DemoqaProject/tests/CheckBoxTest.cs

```diff
@@ -0,0 +1,28 @@
+using DemoqaProject.PageObjects;
+using NUnit.Framework;
+
+namespace DemoqaProject
+{
+    public class CheckBoxTest : BaseSeleniumTest
+    {
+        [Test]
+        public void SelectAll()
```

Comment:
Please add proper name and description for test. It's not clear what  you're testing here.

Reply:
done